sql/hive/src/main/scala/org/apache/spark/sql/hive/client/package.scala

```diff
@@ -102,7 +102,7 @@ package object client {
 
     // Since HIVE-14496, Hive materialized view need calcite-core.
     // For spark, only VersionsSuite currently creates a hive materialized view for testing.
-    case object v2_3 extends HiveVersion("2.3.7",
+    case object v2_3 extends HiveVersion("2.3.8",
```

Comment:
Per local test, this needs to be changed to pass all tests. But let's see the Jenkins result first.